app/api/tickets.py

```diff
@@ -16,20 +18,25 @@ class Meta:
         self_view_kwargs = {'id': '<id>'}
         inflect = dasherize
 
+    @validates_schema
+    def validate_date(self, data):
+        if data['sales_starts_at'] >= data['sales_ends_at']:
+            raise UnprocessableEntity({'pointer': 'sales_ends_at'}, "sales_ends_at should be after sales_starts_at")
+
     id = fields.Str(dump_only=True)
     name = fields.Str(required=True)
     description = fields.Str()
     type = fields.Str(required=True)
-    price = fields.Float()
-    quantity = fields.Integer()
+    price = fields.Float(validate=lambda n: n >= 0)
+    quantity = fields.Integer(validate=lambda n: n >= 0)
     description_toggle = fields.Boolean(default=False)
     position = fields.Integer()
     is_fee_absorbed = fields.Boolean()
-    sales_starts_at = fields.DateTime()
-    sales_ends_at = fields.DateTime()
+    sales_starts_at = fields.DateTime(required=True)
+    sales_ends_at = fields.DateTime(required=True)
     is_hidden = fields.Boolean(default=False)
-    min_order = fields.Integer()
-    max_order = fields.Integer()
+    min_order = fields.Integer(validate=lambda n: n >= 0)
```

Comment:
Is it possible to check and ensure this is lower (or) equal to the `max_order` ?